R/mclosest.R

```diff
@@ -0,0 +1,85 @@
+#'@title Extract closest values in a pairwise manner between two matrices
+#'
+#' @description
+#'
+#' `mclosest` functions calculate the closest rows in a pairwise manner.
+#' It returns the index of the closest row in 'table' for each row in 'x'. 
+#'
+#' @param x `numeric`  matrix or data frame representing the query data. each 
```

Comment:
Some smaller things here: an additional whitespace between `numeric` and matrix. Then I would start the new sentence with *Each row in `x` will be compared to every row in `table`. Both `x` and `table` are expected to have the same number of columns and columns are expected to be in the same order.*

Reply:
done